quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakSessionFactoryPreInitBuildItem.java

```diff
@@ -17,11 +17,45 @@
 
 package org.keycloak.quarkus.deployment;
 
-import io.quarkus.builder.item.EmptyBuildItem;
+import io.quarkus.builder.item.SimpleBuildItem;
+import org.keycloak.provider.Provider;
+import org.keycloak.provider.ProviderFactory;
+import org.keycloak.provider.Spi;
+import org.keycloak.theme.ClasspathThemeProviderFactory;
+
+import java.util.List;
+import java.util.Map;
 
 /**
  * A symbolic build item that can be consumed by other build steps when the {@link org.keycloak.quarkus.runtime.integration.QuarkusKeycloakSessionFactory}
  * is pre-initialized.
  */
-public final class KeycloakSessionFactoryPreInitBuildItem extends EmptyBuildItem {
+public final class KeycloakSessionFactoryPreInitBuildItem extends SimpleBuildItem {
+    private Map<Spi, Map<Class<? extends Provider>, Map<String, Class<? extends ProviderFactory>>>> factories;
+    private Map<Class<? extends Provider>, String> defaultProviders;
```

Comment:
It doesn't look like the fields other than the factories are included. Do you think they will be used later?